### DISCUSSION TITLE
How to enter commands like CTRL R

### DISCUSSION THREAD

**Discussion by mintgitb1ker** [StreamController/StreamController#204] [General]
Any simple user guide I am missing?

**Comment by Core447** [comment]
You can use the "Hotkey" action of the OS plugin for this.

**Reply by mintgitb1ker** [reply]
> You can use the "Hotkey" action of the OS plugin for this.

Thx so much - its about all I use my streamdeck for

**Comment by Core447** [comment]
> Any simple user guide I am missing?

Unfortunately the only guide-like thing we have is [this video](https://youtu.be/kIJOj_6Jimk).

**Reply by Core447** [reply]
@mintgitb1ker And you can also join our [Discord](https://discord.gg/MSyHM8TN3u) for help
